internal/tracker/tracker.go

```diff
@@ -63,6 +91,28 @@ func newTracker(input []byte) (t Tracker, err error) {
 	return
 }
 
+func validateBundleSchema(bundleConfig []byte) error {
+	var sch *jsonschema.Schema
+	sch, err := jsonschema.CompileString("inline", jsonSchema)
+	if err != nil {
+		return err
+	}
+	var t map[string]interface{}
+	if err := yaml.Unmarshal(bundleConfig, &t); err != nil {
+		return err
+	}
+	val, ok := t["trusted_tasks"]
+	if !ok {
+		return fmt.Errorf("the data does not contain a trusted_tasks key")
+	}
+
+	if err = sch.Validate(val); err != nil {
+		return err
```

Comment:
I'm wondering, in lieu of https://github.com/enterprise-contract/ec-policies/pull/1225 if we should warn that there are duplicates?